core/templates/dev/head/pages/exploration_player/ExplorationPlayerStateService.js

```diff
@@ -95,51 +95,45 @@ oppia.factory('ExplorationPlayerStateService', [
       getCurrentStateName: function() {
         return _currentEngineService.getCurrentStateName();
       },
-      getCurrentInteraction: function() {
-        return _currentEngineService.getCurrentInteraction();
-      },
       getNextInteraction: function() {
```

Comment:
Many of these functions should be removed and just called from nextCard.

Reply:
Yeah, many are called like that, forgot to remove it from here, will remove it. Also, interaction instructions are not present in card object currently (mentioned in previous comment).

Reply:
Done.